Fhi.HelseId.Integration.Tests/HelseId.Api/Tests/DPoPTests.cs

```diff
@@ -0,0 +1,99 @@
+using Fhi.ClientCredentialsKeypairs;
+using Fhi.HelseId.Api;
+using Fhi.HelseId.Api.ExtensionMethods;
+using Fhi.HelseId.Integration.Tests.HelseId.Api.Setup;
+using Fhi.HelseId.Integration.Tests.TestFramework;
+using System.Net;
+using System.Text.Json;
+
+namespace Fhi.HelseId.Integration.Tests.HelseId.Api.AppInitiatedTests;
+
+/// <summary>
+/// Need to go through if is as intended
+/// </summary>
+public class DPoPTests 
```

Comment:
De tre testene under er veldig like.  Om du parametriserer dem så blir det enklere, og lettere å se hva som varierer her.  Du kan evt også anvende SetUp for noe av det som er felles.

Reply:
Jeg er personlig ikke så glad i felles setup, men heller hjelpe metoder som tydliggjør oppsettet. Jeg har ikke skrevet testene i utgangspunktet. Det er en liten forbedring og kan helt klart forbedres mer. Jeg tar det med inn i neste oppgave.